pom.xml

```diff
@@ -14,7 +14,7 @@
 
     <properties>
         <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
-        <jenkins.version>1.612</jenkins.version>
+        <jenkins.version>2.7.1</jenkins.version>
```

Comment:
@samrocketman we need to bump the required Jenkins version here to get the newer `User.getUserById()` api